spring-data-jdbc/src/main/java/org/springframework/data/jdbc/repository/support/JdbcRepositoryFactoryBean.java

```diff
@@ -101,13 +102,25 @@ public void setDataAccessStrategy(DataAccessStrategy dataAccessStrategy) {
 		this.dataAccessStrategy = dataAccessStrategy;
 	}
 
+	/**
+	 * @param queryMappingConfiguration can be {@literal null}. {@link #afterPropertiesSet()} defaults to
+	 *          {@link QueryMappingConfiguration#EMPTY} if {@literal null}.
+	 */
+	@Autowired(required = false)
```

Comment:
New members should get a `@since` tag. `@Autowired` on a bean class is asking for trouble. Users are not able to override the annotation while factory beans can always perform a bean lookup themselves within `afterPropertiesSet` while being `BeanFactoryAware`.

Reply:
Nit: If `queryMappingConfiguration` can be `null`, it should be annotated with `@Nullable`.